### DISCUSSION TITLE
今天ALIST突然用不了了！？

### DISCUSSION THREAD

**Discussion by Leon66666666** [AlistGo/alist#4116] [Q&A]
前几天更新最新版3.15.1 ，  用了几天没问题 ，然后今天突然打不开ALIST网页，重新CMD安装出现如下代码，请大神解惑：

Microsoft Windows [版本 10.0.22621.1413]
(c) Microsoft Corporation。保留所有权利。

D:\alist>alist server
INFO[2023-04-11 17:53:44] reading config file: data\config.json
INFO[2023-04-11 17:53:44] load config from env with prefix: ALIST_
INFO[2023-04-11 17:53:44] init logrus...
INFO[2023-04-11 17:53:44] start server @ 0.0.0.0:5244
INFO[2023-04-11 17:53:48] Aria2 not ready.
INFO[2023-04-11 17:53:49] qbittorrent not ready.

**Comment by xhofe** [comment]
这只是启动正常的log，无法说明任何问题。

**Reply by Leon66666666** [reply]
> 这只是启动正常的日志，无法说明任何问题。

问题是我现在运行ALIST打不开  只有这个信息

**Reply by Leon66666666** [reply]
> 这只是启动正常的日志，无法说明任何问题。

请问种情况是ALIST出现问题了还是我的电脑出现问题了？

**Reply by xhofe** [reply]
无法基于你提供的现有信息作出判断，但是一般来说都是使用上的问题。
